deployment/clouddeploy/gke-workers/base/nvd-mirror.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: nvd-mirror
+spec:
+  schedule: "* */2 * * *"
+  concurrencyPolicy: Forbid
+  failedJobsHistoryLimit: 3
+  jobTemplate:
+    spec:
+      activeDeadlineSeconds: 7200
+      template:
+        spec:
+          tolerations:
+          - key: workloadType
+            operator: Equal
+            value: highend
+          containers:
+          - name: nvd-mirror
+            env:
+            - name: WORKDIR
```

Comment:
This is `WORK_DIR` in the script

Reply:
Thank you! Fixed in [0215350](https://github.com/google/osv.dev/pull/1839/commits/0215350534c233f743f5332e10486892ee32e3fa)